generator/src/lib.rs

```diff
@@ -1292,7 +1292,24 @@ pub fn derive_setters(_struct: &vkxml::Struct) -> Option<Tokens> {
         _ => None,
     });
 
+    let count_members: Vec<String> = members.clone().filter_map(|field| {
+        if field.array.is_some() {
+            if let Some(ref array_size) = field.size {
+                if !array_size.starts_with("latexmath") && array_size.ends_with("Count") {
```

Comment:
What invalid cases are excluded by the `ends_with("Count")` constraint?

Reply:
That's a failure in my brain. Thanks for catching that.